vault/datadog_checks/vault/metrics.py

```diff
@@ -37,6 +37,20 @@
     'process_start_time_seconds': 'process.start_time.seconds',
     'process_virtual_memory_bytes': 'process.virtual_memory.bytes',
     'process_virtual_memory_max_bytes': 'process.virtual_memory.max.bytes',
+    'secrets_pki_tidy_cert_store_current_entry': 'secrets.pki.tidy.cert_store_current_entry',
+    'secrets_pki_tidy_cert_store_deleted_count': 'secrets.pki.tidy.cert_store_deleted_count',
+    'secrets_pki_tidy_cert_store_total_entries': 'secrets.pki.tidy.cert_store_total_entries',
+    'secrets_pki_tidy_cert_store_total_entries_remaining': 'secrets.pki.tidy.cert_store_total_entries_remaining',
+    'secrets_pki_tidy_duration': 'secrets.pki.tidy.duration',
+    'secrets_pki_tidy_failure': 'secrets.pki.tidy.failure',
+    'secrets_pki_tidy_revoked_cert_current_entry': 'secrets.pki.tidy.revoked_cert_current_entry',
+    'secrets_pki_tidy_revoked_cert_deleted_count': 'secrets.pki.tidy.revoked_cert_deleted_count',
+    'secrets_pki_tidy_revoked_cert_total_entries': 'secrets.pki.tidy.revoked_cert_total_entries',
+    'secrets_pki_tidy_revoked_cert_total_entries_fixed_issuers': 'secrets.pki.tidy.revoked_cert_total_entries_fixed_issuers',
+    'secrets_pki_tidy_revoked_cert_total_entries_incorrect_issuers': 'secrets.pki.tidy.revoked_cert_total_entries_incorrect_issuers',
```

Comment:
These lines are too long for the linter. I'd rather not break the style of the rest of the dict to make the linter happy. Can I add `# noqa: E501`? If not, how would you like this formatted?

Reply:
`  # noqa: E501` is fine!